app/models/labware.rb

```diff
@@ -29,7 +29,16 @@ class Labware < Asset
   has_many :requests_as_target, through: :receptacles
   has_many :transfer_requests_as_source, through: :receptacles
   has_many :transfer_requests_as_target, through: :receptacles
+
+  # Submissions in progress for the labware. Found by looking at those associated
+  # with the transfer requests into the receptacles. Be a little cautious using this,
+  # as it will not handle cross-submission pools, but is better for historical
+  # data.
   has_many :submissions, through: :receptacles
+
+  # Direct submissions are those made on the plate itself, and are found via
+  # the orders associated with the well.s
```

Comment:
```suggestion
  # the orders associated with the wells
```

Reply:
I'll wrap this into my doc changes to avoid triggering the CI too much.